gnovm/tests/files/typeassert11.gno

```diff
@@ -0,0 +1,9 @@
+package main
+
+type zilch interface{}
+
+func main() {
+	_ = zilch(nil)
+}
+
+// Output:
```

Comment:
how about just print something to indicate it's working? like `ok` 😆

Reply:
add some fancy print 
https://github.com/gnolang/gno/pull/2699/commits/9b8f6971131523a4af78ca647ad8c3d7ee5c1319